tests/test_api.py

```diff
@@ -42,20 +43,17 @@ def test_api(provider, network, kwargs, si_units, station_id):
     """Test main wetterdienst API"""
     # Build API
     api = Wetterdienst(provider, network)
-
+    if api == HubeauRequest:
```

Comment:
Not sure?
```suggestion
    if api is HubeauRequest:
```

Reply:
Actually I changed it and `pytest.param(..., marks=pytest.mark.xfail())` is cleaner in this case.